provider/virtio_fs_utils.py

```diff
@@ -0,0 +1,211 @@
+import logging
+import os
+
+from avocado.utils import process
+
+from virttest import data_dir
+from virttest import error_context
+from virttest import utils_misc
+
+LOG_JOB = logging.getLogger('avocado.test')
+
+
+def get_virtiofs_driver_letter(test, fs_target, session):
+    """
+    Get the virtiofs driver letter( Windows guest only )
+    :param test: QEMU test object
+    :param fs_target: virtio fs target
+    :param session: the session of guest
+    :return driver_letter: the driver letter of the virtiofs
+    """
+    error_context.context("Get driver letter of virtio fs target, "
+                          "the driver label is %s." % fs_target, LOG_JOB.info)
+    driver_letter = utils_misc.get_winutils_vol(session, fs_target)
+    if driver_letter is None:
+        test.fail("Could not get virtio-fs mounted driver letter.")
+    return driver_letter
+
+
+def basic_io_test(test, params, session):
+    """
+    Virtio_fs basic io test. Create file on guest and then compare two md5
+    values from guest and host.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param session: the session from guest
+    """
+    error_context.context("Running viofs basic io test", LOG_JOB.info)
+    test_file = params.get('virtio_fs_test_file', "virtio_fs_test_file")
+    cmd_dd = params.get("virtio_fs_cmd_dd", 'dd if=/dev/urandom of=%s bs=1M '
+                                            'count=100 iflag=fullblock')
+    windows = params.get("os_type", "windows") == "windows"
+    io_timeout = params.get("timeout", 120)
```

Comment:
```suggestion
    io_timeout = params.get("fs_io_timeout", 120)
```
The `timeout` is a bit common.